texturepacks/RemisTexturePack/_meta_json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Remi's texture pack",
+  "author": "Remi",
+  "description": "Such a cool pack!"

```

Comment:
can you change this to be a bit more descriptive?

Reply:
wdym a bit more descriptive?

Reply:
what should i do?

Reply:
describe what the pack actually adds or changes

Reply:
kk

Reply:
you didnt edit the file